openshift/release/update-to-head.sh

```diff
@@ -16,6 +16,10 @@ openshift/release/mirror-upstream-branches.sh
 git fetch upstream main
 git checkout upstream/main -B ${REPO_BRANCH}
 
+# Remove GH Action hooks from upstream
+rm -rf .github/workflows
+git commit -sm ":fire: remove unneeded workflows" .github/
```

Comment:
Sorry for my ignorance of the git command. But why does `-s` option is used instead of `-a`?

Reply:
Never mind. I understand.

Reply:
`-a` does not work as the file is removed. With said, I wonder `-s` is not necessary. What whose `sign-off` will be added :thinking: Let's see tomorrow's commit...